src/widget/OktaSignIn.js

```diff
@@ -92,10 +138,32 @@ var OktaSignIn = (function () {
     };
   }
 
+  function checkResponseVersion (response) {
+    response.version = '1.0.0';
```

Comment:
If this is temporary, please add a // TODO comment

Reply:
Oops that will be removed. For limited EA this version we publish will be `1.0.0`. But I just learned that going forward we will be following a semantic version for this API and any version >= `1.0.0` means we are on the new pipeline. The old pipeline API will also get a version.